daemons/controld/controld_corosync.c

```diff
@@ -59,6 +59,17 @@ crmd_cs_dispatch(cpg_handle_t handle, const struct cpg_name *groupName,
             crm_update_peer_proc(__func__, peer, crm_proc_cpg,
                                  ONLINESTATUS);
         }
+
+	// Solution One (the poke comes from crmd::peer_update_callback)
+	if (pcmk__str_eq(crm_element_value(xml, T_CRM_OPERATION), "poke", TRUE)) {
+	    return;
+	}
+
+	// Solution Two (the poke comes from fenced::st_peer_update_callback)
+	if (pcmk__str_eq(crm_element_value(xml, "st_op"), "poke", TRUE)) {
+	    return;
+	}
+
```

Comment:
The controller won't get any CPG messages from the fencer, so you don't need solution 2.
Messages aren't actually handled here, but a little further in the call chain in handle_request() (when the message was created by create_request()).

Reply:
Ok.